packages/pigeon/lib/cpp_generator.dart

```diff
@@ -656,7 +656,7 @@ $prefix\t}${indent.newline}''';
               if (method.isAsynchronous) {
                 methodArgument.add(
                   '[&wrapped, &reply]($returnTypeName&& output) {${indent.newline}'
-                  '${wrapResponse('\treply(wrapped);${indent.newline}', method.returnType)}'
+                  '${wrapResponse('\treply(flutter::EncodableValue(wrapped));${indent.newline}', method.returnType)}'
```

Comment:
We should be able to `flutter::EncodableValue(std::move(wrapped))` here, so we're not copying `wrapped`. We don't even use `wrapped` again after calling `reply` anywhere, right?